WSL/install-win10.md

```diff
@@ -1,40 +1,48 @@
 ms.topic: article
 ms.localizationpriority: high
 ---
 
 # Windows Subsystem for Linux Installation Guide for Windows 10
 
-## Install the Windows Subsystem for Linux
+## Install Windows Subsystem for Linux
 
-Before installing any Linux distributions on Windows, you must enable the "Windows Subsystem for Linux" optional feature.
+Windows Subsystem for Linux has two different versions to choose between during the installation process. WSL 2 has better overall performance and we recommend using it. If your system does not support WSL 2, or you have a specific situation that requires cross-system file storage, then you may want to stick with WSL 1. Read more about [Comparing WSL 2 and WSL 1](./compare-versions.md).
+
+## Step 1 - Enable the Windows Subsystem for Linux
+
+You must first enable the "Windows Subsystem for Linux" optional feature before installing any Linux distributions on Windows.
 
 Open PowerShell as Administrator and run:
 
 ```powershell
 dism.exe /online /enable-feature /featurename:Microsoft-Windows-Subsystem-Linux /all /norestart
 ```
 
-To only install WSL 1, you should now restart your machine and move on to [Install your Linux distribution of choice](./install-win10.md#install-your-linux-distribution-of-choice), otherwise wait to restart and move on to update to WSL 2. Read more about [Comparing WSL 2 and WSL 1](./compare-versions.md).
+We recommend now moving on to step #2, updating to WSL 2, but if you wish to only install WSL 1, you can now restart your machine and move on to [Step 6 - Install your Linux distribution of choice](./install-win10.md#step-6---install-your-linux-distribution-of-choice). To update to WSL 2, wait to restart your machine and move on to the next step.
+
+## Step 2 - Update to WSL 2
 
-## Update to WSL 2
+To update to WSL 2, you must be running Windows 10.
 
-To update to WSL 2, you must meet the following criteria:
+### Requirements
 
-- Running Windows 10, [updated to version 1903 or higher](ms-settings:windowsupdate), **Build 18362** or higher for x64 systems.
-   - If you are on Windows 10 version 1903 or 1909 make sure your minor build number is 1049 or higher. See [full troubleshooting instructions here](https://docs.microsoft.com/windows/wsl/troubleshooting#im-on-windows-10-version-1903-and-i-still-do-not-see-options-for-wsl-2)
-- Running Windows 10, updated to version 2004 or higher, **build 19041**, for ARM64 systems.
-- Please note if you are on Windows 10 version 1903 or 1909 you will need to ensure that you have the proper backport, instructions can be [found here](https://devblogs.microsoft.com/commandline/wsl-2-support-is-coming-to-windows-10-versions-1903-and-1909/#how-do-i-get-it). 
+- For x64 systems: **Version 1903** or higher, with **Build 18362** or higher.
+- For ARM64 systems: **Version 2004** or higher, with **Build 19041** or higher.
+- Builds lower than 18362 do not support WSL 2. Use the [Windows Update Assistant](https://www.microsoft.com/software-download/windows10).
```

Comment:
@mattwojo It says 'Use the UPdate Assistant', but to do what? To update?

Reply:
I'll update to: "Use the Windows Update Assistant to update your version of Windows." Sound ok?